### ISSUE TITLE
I can't log in to my Yetiforce database

### ISSUE THREAD

**Issue by asperitigroup** [YetiForceCompany/YetiForceCRM#13518]
My login page opens, but when I put in my username and password, nothing happens, it just loops back to the login page - no messages. I have tried restoring the database prior to the issue occurring, but it is still the same problem, can't log in.

**Comment by mariuszkrzaczkowski** [comment]
![github](https://user-images.githubusercontent.com/10198654/93861747-513c1080-fcc1-11ea-8bea-ca91a54c83c1.jpg)

**Comment by asperitigroup** [comment]
I edited the debug.php file to log errors.

class Debug
{
	/** Enable saving logs to file. Values: false/true */
	public static $LOG_TO_FILE = true;

	/** Enable saving logs profiling. Values: false/true */
	public static $LOG_TO_PROFILE = false;

	/** Level of saved/displayed logs. Values: false = All / 3 = error and warning / ["error", "warning", "info", "trace", "profile"] */
	public static $LOG_LEVELS = ["error", "warning", "info", "trace", "profile"];

	/** Level of saved/displayed tracerts. // Values: int */
	public static $LOG_TRACE_LEVEL = 9;

	/** Display main debug console */
	public static $DISPLAY_DEBUG_CONSOLE = false;

	/** Enable displaying logs in debug console. Values: false/true */
	public static $DISPLAY_LOGS_IN_CONSOLE = false;

I then opened the login page again at https://asperiti.uk/yetiforce

I could not log in again, it just reloaded the login page. 

I then looked in the cache/logs folder - no new logs have been created, just the old ones from the original install.

![image](https://user-images.githubusercontent.com/58600791/93893056-1e0b7880-fce5-11ea-94a4-74fbae2825fb.png)

I can't get in to my system to show the Yetiforce server configuration as I can't log in.

Sorry, I'm trying to provide as much information as I can, but my technical knowledge is limited.

**Comment by mariuszkrzaczkowski** [comment]
check php logs

**Comment by asperitigroup** [comment]
Please excuse my ignorance Marius, but how do I check the php logs? Where do I find the php.ini file to change log-errors and display-errors to on?
This is the first time I have had a technical problem with Yetiforce, so I haven't had to do troubleshooting before, I'm more than happy to provide any needed information if I know how to do it.

**Comment by mariuszkrzaczkowski** [comment]
try it like this
https://www.google.com/search?q=find+php+log+file&oq=find+php+logs

**Comment by asperitigroup** [comment]
OK, I created a phpinfo.php file, and ran it in the root of my webspace, and it showed me this, how do I now edit it?

![image](https://user-images.githubusercontent.com/58600791/93897339-de935b00-fce9-11ea-9c77-381efdf8f3c4.png)

**Comment by asperitigroup** [comment]
I'm not sure what is going on here. When I tried to upload the phpinfo.php file that I had created, my FTP program wouldn't upload it. It had a File transfer failed. When I then went in to my hosts file manager and tried to upload it, it said I did not have enough space - although I have only used 25Gb out of my 100GB storage.

![image](https://user-images.githubusercontent.com/58600791/93897982-9de81180-fcea-11ea-8015-cde95a381e9b.png)

So I deleted a file of about 5Mb that I did not need, and was able to upload the phpinfo.php file and run it.

Then I went back to my Yetiforce login page and tried to login again. AND IT WORKED!

So it looks like the amount of free space in my host storage is being mis-reported in some way.

I am going to ask 1and1 Ionos support what is happening, but for now it appears to be fixed.

MANY THANKS FOR YOUR HELP, I'm glad it was not a Yetiforce issue.
